### DISCUSSION TITLE
你好，请问能集成 melt_eng 方案吗？

### DISCUSSION THREAD

**Discussion by moonloadermic** [LufsX/rime#27] [General]
希望优化输入英文的体验，类似于 在 https://www.youdao.com/ 输入 exten 时，有如下图的效果（英语单词翻译可以不要，就是输入英语单词的前几个字母，候选英文单词出现）：

参考：https://dvel.me/posts/make-rime-en-better/

![1](https://github.com/LufsX/rime/assets/130746790/cf2ccfb4-a007-4442-aa83-14d8848a4102)

**Comment by LufsX** [comment]
可以，考虑单独开一个分支，主分支就不集成了

不过先等我忙完这段时间吧 😂

**Comment by moonloadermic** [comment]
谢谢🙏，祝您万事如意😄

**Comment by LufsX** [comment]
已经在分支 [melt-eng](https://github.com/LufsX/rime/tree/melt-eng)  的 https://github.com/LufsX/rime/commit/dd9ecacd271bbd030ede796f5020d70f9802a46c 集成

注意，默认为小鹤双拼的输入方案，其它的需要手动按 [iDvel/rime-ice](https://github.com/iDvel/rime-ice) 内相关说明修改

**Comment by moonloadermic** [comment]
再次感谢！元旦快乐！

**Reply by LufsX** [reply]
这个分支会不定期的同步主分支的内容，我用了一番应该没啥问题

元旦快乐～
